main.go

```diff
@@ -266,13 +284,6 @@ func runCommand(command flags.Commander, args []string) error {
 }
 
 func checkPrerequisites() error {
-	if !nbd.IsSupported() {
-		return errors.New("cannot create nbd devices (is qemu-nbd installed?)")
-	}
-	if !nbd.IsModuleLoaded() {
-		return errors.New("cannot create nbd devices because the required kernel module is not loaded")
-	}
```

Comment:
Goot catch - put the check back. But I decided to write a dedicated NBD pre-reqs check function and run it conditionally for block devices and unconditionally for file path case. Pls let me know if you see a better way to do that check.

Reply:
Hey @chrisccoulson, could you review the new changes when you find a minute? I also added a version to the binary so that we know what version we are running. Let me know if that's how you guys do it. Thanks.